### ISSUE TITLE
Connect metatrader4 with IQ-Option

### ISSUE THREAD

**Issue by yesamine** [n1nj4z33/iqoptionapi#56]
Please i want to connect metatrader4 with IQ-Option for built auto trading bot and profit from the awesome indicator and signals in metatrader4 

please can help me how i can do this
how i can connect wss in mt4 to IQ
mt4 have a libe for wss or not ?? if not how i can add one to it
mt4 support python ?? if yes how i can merge this python api with mt4 expert 

please if any one can give me some help and some information to be in the good way

@n1nj4z33 @fsnlarson @boskiv @freenetwork @PerryAwesome @lucianopinheiro @m10141 @NazirH @EVAZA @Niks005 @freeost @cz3kit  @vinydl @leonardomutti @Aiacos @playsten4096 @CZ-DannyK @l9sheen @SonyPony @kanXuT @joelmm1 @MadOne199 @Shirokr @kiddystarnova @HubSpace2000  @frxncisjoseph @rafinha90 @genesiscrew  @joelmm1 @Olascoaga @gocholo5 @engelkohle @boskiv @antoinejoerg  @Amsori  @crypto-maniac @BremerBlizzard @j1and1  @SiliconDare @rimalroc @AllenUchiha @burny91  @santiagonn @0xAalaoui @Aiacos @shiundu  @joebrassi  

thanks you all

**Comment by EVAZA** [comment]
http://www.mt2iq.com

**Comment by yesamine** [comment]
@EVAZA Price: 300 USD

**Comment by yesamine** [comment]
thank you @EVAZA for www.mt2iq.com solution its very good 

but really i dont have money to buy this solution because i lose my job and im a father of 2 kids and you know living expenses..... , thats way i try to find a free solution or coded by my self

please if you can give me help i never forget your kindness

thank you @EVAZA

**Comment by gelsas** [comment]
Is there any other solution then mt2iq ?

**Comment by EVAZA** [comment]
No. MT2IQ is the best solution I know. Other solutions are just clickers. If you are looking for a bot that you can operate through MetaTrader, so you are right with MT2IQ. There is a indicator connector and library. The library is very usefull if you want to place trades directly from MQL code. So you need just to call a function: `mt2iq(Symbol(), "CALL", Amount, ExpiryTime)`

**Comment by akashva50** [comment]
@yesamine . Do you find any solution so far. I like the way mt2iq works. Awesome bot. But its price is too
high. Any way to crack it or recode it for removing initial authentication. Can we hack the trial version to 
use full version. Am doing some research.

**Comment by playsten4096** [comment]
Hi i have made my own version and it works well. But i wouldnt sell it less
than 500 USD.

Kind Regards

2018-01-24 17:19 GMT+07:00 akashva50 <notifications@github.com>:

**Comment by NazirH** [comment]
I don't mind paying the amount as long as it works. First I need to test it
out for a month on the demo account. If it works as you claim, I will pay
you from the first $500 I make using your application. Let us discuss
detail to proceed.

Awaiting your reply
Regards
Nazir Hajje

**Comment by akashva50** [comment]
What is the need of buying one for $500. Mt2iq costs only $300.

**Comment by Amsori** [comment]
Hello,
Can i use the code to integrated with vb.net programming ?

Pada tanggal 30 Jan 2018 21:36, "akashva50" <notifications@github.com>
menulis:

What is the need of buying one for $500. Mt2iq costs only $300.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/n1nj4z33/iqoptionapi/issues/56#issuecomment-361611694>,
or mute the thread

**Comment by yesamine** [comment]
i have a good solution best than all this 
the solution is very simple to link with mt4
the bot have :
- Working time and work session for trading , easy to manage
- Good Money managment (double the wining not the martingal) easy to manage with multi options like stopLose
- Strategies : multiple Type of strategies can work with
you can merge indicators easelly in the bot 
you can set indicators for trad and others for validation

if you wont more information plz contact me in private

**Comment by akashva50** [comment]
How @yesamine

**Comment by yesamine** [comment]
i developpe this

**Comment by yesamine** [comment]
hello.fethallah@gmail.com

**Comment by Rooks25** [comment]
Recently I have tested Both MT2IQ and the Bot developed by Yesamine, IQ Bot 3.2
Really has interesting features to linking MT4 indicators to IQ Option.
You can manage strategies with several indicators that MT2IQ cant. MT2IQ only works connecting one indicator to put the trade. With IQ Bot 3.2 you can manage a combination of indicators to result an strategy that really hits good winning percentages.
The moneymanagement has an interesting feature to double the winning price as you realise consecutive winning trades.The possibilty to add indicators as you choose to connect from MT4 to IQ Option and manage them to interact between their signals adds an infinite world of possibilities that MT2IQ don´t have.
Here´s a link to video showinf its features:
[https://youtu.be/fzW8eqsJ_3M](url)

**Comment by yesamine** [comment]
thank you so mutch
really i do my effort to complet your expiry time algorithme
i just have one problem that the father of my wife in hospital he has heart
problems
please give me some time

2018-02-14 1:54 GMT+00:00 Rooks25 <notifications@github.com>:

**Comment by yesamine** [comment]
the Version now is 3.3 :+1:

**Comment by EVAZA** [comment]
Locate the application that is giving you “CreateProcess Failed Code 740” error and right click it. Choose Properties from the list. When Properties window opens, navigate to the Compatibility tab. Make sure that you check “Run this program as an administrator”

**Comment by azartheen** [comment]
![screenshot 20](https://user-images.githubusercontent.com/31647728/39012825-274f0f5a-4433-11e8-85a8-88ae5be574f8.png)

i purchase from online but there is a problem.... what a shit is happening.... it is not a good thing... help me.... i spend my money.... what is the prob

**Comment by Rooks25** [comment]
You have purchased the software with the same mail account that IQ Option? 
If you have misspelled any letter in purchase, you will not be able to use the software... 

⁣Enviado desde mi Android. ​

**Comment by master88** [comment]
Can you give me try you version 3.3 😍

**Comment by yesamine** [comment]
current version is 3.7

contact me by mail: hello.fethallah@gmail.com

2018-05-06 23:12 GMT+00:00 master88 <notifications@github.com>:

**Comment by azartheen** [comment]
@master88 why you are asking version 3.3.... it is comfortable or else... contact me on asartheen916@gmail.com

**Comment by djpaulomix1** [comment]
versão 3.3
DJPAULOMIX1@GMAIL.COM

**Comment by azartheen** [comment]
Why most peoples ask mt2iq version 3.3????

**Comment by iQBotProject** [comment]
> **almost of people ask me to post more information about the robot**

![iq](https://user-images.githubusercontent.com/32252265/65209769-f4e05880-da90-11e9-8e40-aa6592040eb8.PNG)

**iQBot** is your way to automate any mt4 strategy indicators for auto-trading binary trading
robot help you with advanced tools and easy methods to use.

- iQBot allowing you to use multi indicators at the same time also multi timeframes
- with multi indicators you can build a complex strategy as you like with iQBot concept
- you can setup multi strategies and use all

> for each strategy, you can link money management, trading control, and active sessions.

- advance **money management** has everything stop loss take profit with a fixed amount or % of the balance, recover the losses with multi-options also double the winning, or you can merge everything.
- **trading control** allows you to manage your trading way and set your conditions.
- **active sessions** to set up the trading time with pair also so advanced 

> Is also multi money-management, multi trading-control, multi active-sessions


using **Backtest** now is super easy with iQBot to test your real strategy performance
just run the robot on backtest mode to go at the fast speed testing your strategy or strategies at one time.
also, use **Strategies Generator** for making hundreds of strategies from some indicators and test all on backtest at the same time testing

**_contact info_**
website : https://iqbotproject.com
whatsapp: https://wa.me/212674429706
chat: https://m.me/binary.iqbot
email: ultra.smart.bot@gmail.com
page : https://www.fb.com/binary.iqbot
group : https://www.fb.com/groups/203883413709307

**Comment by freenetwork** [comment]
Friends, on the territory of Russia IQ is blocked, and the platform does not allow to trade. Development got up a long time ago because of this

**Comment by playsten4096** [comment]
> Friends, on the territory of Russia IQ is blocked, and the platform does not allow to trade. Development got up a long time ago because of this

Yes, because "people" who "live" in russia in 2019 = are stupid slaves and they dont deserve to trade on world markets) They must work for 250$ per month and love putin) Рабы ебучие) Скот

**Comment by kananiviral56-hub** [comment]
i pay all money when i earn 300$ from your indicator then only i interested or you may gave me signal till i made 300$ , then after i purchase from you and thats how i think.
